examples/rsp-cra-18/src/BodyContent.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
```

Comment:
```suggestion
import React, {FormEvent, Key, useRef, useState} from "react";
```
you could do something like the above so you don't need all the `React.` below